### DISCUSSION TITLE
Install Maven and Generate Mapper

### DISCUSSION THREAD

**Discussion by moriyahazan** [developers-bootcamp/order-manager-red#16] [General]
In order to generate a mapper by Mapstruct: 

- install maven: https://phoenixnap.com/kb/install-maven-windows
- run `mvn clean install `
- verify mapper has implementation with mapping methods

**Comment by moriyahazan** [comment]
@developers-bootcamp/green 
@developers-bootcamp/red  

Hi Girls, 

I got several questions about Maven

please reply on this chat if you install Maven and generate Mapper or not
if not please write me what the problem 

I am waiting for notifications from all of you

thanks
Moriya

**Comment by ShirelAs** [comment]
התקנתי את ה maven אבל הוא לא מזהה אותו ולא יוצר את ה DTO



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03.07.23,
15:26:17

‫בתאריך יום ב׳, 3 ביולי 2023 ב-15:25 מאת ‪Moriya‬‏ <‪
***@***.***‬‏>:‬

**Reply by moriyahazan** [reply]
Did you add it to environment variables?

**Comment by RutiChuri** [comment]
התקנתי את MAVEN ומראה שהוא לא מוצא אותו וגם הפרויקט לא עובד בצורה נכונה
בעקבות זה.

‫בתאריך יום ב׳, 3 ביולי 2023 ב-15:25 מאת ‪Moriya‬‏ <‪
***@***.***‬‏>:‬

**Comment by ShirelAs** [comment]
כן
תודה וסליחה על ההפרעה



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03.07.23,
15:31:27

‫בתאריך יום ב׳, 3 ביולי 2023 ב-15:30 מאת ‪Moriya‬‏ <‪
***@***.***‬‏>:‬

**Comment by RivkaEhrenfeld** [comment]
לא התקנתי משהו מיוחד


‫בתאריך יום ב׳, 3 ביולי 2023 ב-15:33 מאת ‪Shirel Asayag‬‏ <‪
***@***.***‬‏>:‬

**Comment by RutiChuri** [comment]
הי מוריה זה רותי חורי אני חייבת עזרה בדחיפות.

‫בתאריך יום ב׳, 3 ביולי 2023 ב-15:50 מאת ‪RivkaEhrenfeld‬‏ <‪
***@***.***‬‏>:‬

**Comment by Elisheva1234** [comment]
את התקנת לי וזה לא עזר אני לא מצליחה לעשות ולא מבינה את השגיאה
[image: image.png]


‫בתאריך יום ג׳, 4 ביולי 2023 ב-10:56 מאת ‪Ruti Churi‬‏ <‪
***@***.***‬‏>:‬
